--- ref
+++ di
@@ -65,23 +65,8 @@
     DTD 2:  1366x768    59.789541 Hz 683:384   47.712 kHz     85.500000 MHz
                  Hfront   70 Hsync 143 Hback  213 Hpol P
                  Vfront    3 Vsync   3 Vback   24 Vpol P
-Checksum: 0xab  Unused space in Extension Block: 105 bytes
+Checksum: 0xab
 
 ----------------
 
-Warnings:
-
-Block 0, Base EDID:
-  Basic Display Parameters & Features: sRGB is signaled, but the gamma != 2.2.
-Block 1, CTA-861 Extension Block:
-  IT Video Formats are overscanned by default, but normally this should be underscanned.
-  Display Product Serial Number is set, so the Serial Number in the Base EDID should be 0.
-
-Failures:
-
-Block 0, Base EDID:
-  Basic Display Parameters & Features: sRGB is signaled, but the chromaticities do not match.
-Block 1, CTA-861 Extension Block:
-  Missing VCDB, needed for Set Selectable RGB Quantization to avoid interop issues.
-
-EDID conformity: FAIL
+EDID conformity: PASS
