--- ref
+++ di
@@ -29,9 +29,9 @@
     DMT 0x12:  1024x768    75.028582 Hz   4:3     60.023 kHz     78.750000 MHz
     DMT 0x24:  1280x1024   75.024675 Hz   5:4     79.976 kHz    135.000000 MHz
   Standard Timings:
-    DMT 0x23:  1280x1024   60.019740 Hz   5:4     63.981 kHz    108.000000 MHz
-    DMT 0x33:  1600x1200   60.000000 Hz   4:3     75.000 kHz    162.000000 MHz
-    DMT 0x15:  1152x864    75.000000 Hz   4:3     67.500 kHz    108.000000 MHz
+    DMT 0x23:  1280x1024   60.000000 Hz 5:4
+    DMT 0x33:  1600x1200   60.000000 Hz 4:3
+    DMT 0x15:  1152x864    75.000000 Hz 4:3
   Detailed Timing Descriptors:
     DTD 1:  1920x1200   59.950171 Hz   8:5     74.038 kHz    154.000000 MHz (519 mm x 320 mm)
                  Hfront   48 Hsync  32 Hback   80 Hpol P
