--- ref
+++ di
@@ -19,23 +19,14 @@
     Green: 0.1875, 0.7060
     Blue : 0.1484, 0.0644
     White: 0.3134, 0.3291
-  Established Timings I & II:
-    IBM     :   720x400    70.081663 Hz   9:5     31.467 kHz     28.320000 MHz
-    DMT 0x04:   640x480    59.940476 Hz   4:3     31.469 kHz     25.175000 MHz
-    DMT 0x06:   640x480    75.000000 Hz   4:3     37.500 kHz     31.500000 MHz
-    DMT 0x09:   800x600    60.316541 Hz   4:3     37.879 kHz     40.000000 MHz
-    DMT 0x0b:   800x600    75.000000 Hz   4:3     46.875 kHz     49.500000 MHz
-    DMT 0x10:  1024x768    60.003840 Hz   4:3     48.363 kHz     65.000000 MHz
-    DMT 0x12:  1024x768    75.028582 Hz   4:3     60.023 kHz     78.750000 MHz
-    DMT 0x24:  1280x1024   75.024675 Hz   5:4     79.976 kHz    135.000000 MHz
   Standard Timings:
-    DMT 0x23:  1280x1024   60.019740 Hz   5:4     63.981 kHz    108.000000 MHz
-    DMT 0x33:  1600x1200   60.000000 Hz   4:3     75.000 kHz    162.000000 MHz
-    DMT 0x15:  1152x864    75.000000 Hz   4:3     67.500 kHz    108.000000 MHz
+    DMT 0x23:  1280x1024   60.000000 Hz 5:4
+    DMT 0x33:  1600x1200   60.000000 Hz 4:3
+    DMT 0x15:  1152x864    75.000000 Hz 4:3
   Detailed Timing Descriptors:
     DTD 1:  1920x1200   59.950171 Hz   8:5     74.038 kHz    154.000000 MHz (519 mm x 320 mm)
-                 Hfront   48 Hsync  32 Hback   80 Hpol P
-                 Vfront    3 Vsync   6 Vback   26 Vpol N
+                 Hfront   48 Hsync  32 Hback   80
+                 Vfront    3 Vsync   6 Vback   26
     Display Product Serial Number: 'G283H8BI21MS'
     Display Product Name: 'DELL 2408WFP'
     Display Range Limits:
