--- ref
+++ di
@@ -17,7 +17,6 @@
     Green: 0.2949, 0.6083
     Blue : 0.1533, 0.0605
     White: 0.3134, 0.3291
-  Established Timings I & II: none
   Standard Timings: none
   Detailed Timing Descriptors:
     DTD 1:  2560x1440   59.999068 Hz  16:9     91.979 kHz    257.540000 MHz (309 mm x 173 mm)
@@ -33,9 +32,4 @@
 
 ----------------
 
-Failures:
-
-Block 0, Base EDID:
-  Missing Display Product Name.
-
-EDID conformity: FAIL
+EDID conformity: PASS
