--- ref
+++ di
@@ -18,17 +18,16 @@
     Green: 0.2890, 0.6093
     Blue : 0.1455, 0.0703
     White: 0.3125, 0.3291
-  Established Timings I & II: none
   Standard Timings: none
   Detailed Timing Descriptors:
     DTD 1:  1920x1080  119.982181 Hz  16:9    137.260 kHz    285.500000 MHz (field sequential R/L, 598 mm x 336 mm)
-                 Hfront   48 Hsync  32 Hback   80 Hpol P
-                 Vfront    3 Vsync   5 Vback   56 Vpol P
+                 Hfront   48 Hsync  32 Hback   80
+                 Vfront    3 Vsync   5 Vback   56
     DTD 2:  1920x1080   99.930409 Hz  16:9    113.221 kHz    235.500000 MHz (field sequential R/L, 598 mm x 336 mm)
-                 Hfront   48 Hsync  32 Hback   80 Hpol P
-                 Vfront    3 Vsync   5 Vback   45 Vpol N
+                 Hfront   48 Hsync  32 Hback   80
+                 Vfront    3 Vsync   5 Vback   45
     Display Range Limits:
-      Monitor ranges (GTF): 24-120 Hz V, 26-140 kHz H, max dotclock 300 MHz
+      Monitor ranges: 24-120 Hz V, 26-140 kHz H, max dotclock 300 MHz
     Display Product Name: 'S27A950D'
   Extension blocks: 1
 Checksum: 0x1b
@@ -40,11 +39,4 @@
   Underscans IT Video Formats by default
   Basic audio support
   Native detailed modes: 0
-  Audio Data Block:
-    Linear PCM:
-      Max channels: 2
-      Supported sample rates (kHz): 48 44.1 32
-      Supported sample sizes (bits): 24 20 16
-  Speaker Allocation Data Block:
-    FL/FR - Front Left/Right
-Checksum: 0x71  Unused space in Extension Block: 115 bytes
+Checksum: 0x71
