--- ref
+++ di
@@ -50,15 +50,4 @@
 
 ----------------
 
-Warnings:
-
-Block 0, Base EDID:
-  Standard Timings: Standard Timing 1152x921 has a dubious odd vertical resolution.
-  Detailed Timing Descriptor #1: DTD is similar but not identical to DMT 0x23.
-EDID:
-  Base EDID: Some timings are out of range of the Monitor Ranges:
-    Vertical Freq: 56.250 - 76.000 Hz (Monitor: 56.000 - 75.000 Hz)
-    Maximum Clock: 141.822 MHz (Monitor: 140.000 MHz)
-    Could be due to a Monitor Range off-by-one rounding issue
-
 EDID conformity: PASS
